tests/EdFi.Common.UnitTests/Security/Pbkdf2HmacSha1SecureHasherTests.cs

```diff
@@ -24,39 +26,36 @@ public class When_computing_a_valid_hash : TestFixtureBase
         {
             private PackedHash _actualResult;
             private string _expectedBytes;
-            private SecureHashRequest _request;
             private Pbkdf2HmacSha1SecureHasher _secureHasher;
+            private int _hashAlgorithm;
+            private byte[] _salt;
 
             protected override void Arrange()
             {
-                _secureHasher = new Pbkdf2HmacSha1SecureHasher(null);
+                _secureHasher = new Pbkdf2HmacSha1SecureHasher();
+
+                _hashAlgorithm = HashHelper.GetSha256Hash(Pbkdf2HmacSha1SecureHasher.ConfigurationAlgorithmName).ToInt32();
```

Comment:
Shouldn't this just read from the AlgorithmHashCode property on the hasher?

Reply:
done

Reply:
hmm looks like iterations is 100k instead of 10k, reverted the change